esp/services/ws_fileio/ws_fileioservice.cpp

```diff
@@ -149,7 +181,7 @@ bool CWsFileIOEx::onCreateFile(IEspContext &context, IEspCreateFileRequest &req,
 
     StringBuffer destAbsPath;
     StringBuffer destNetAddr;
-    if (!CheckServerAccess(server, destRelativePath, destNetAddr, destAbsPath))
+    if (!CheckServerAccess(server, req.getDestNetAddress(), destRelativePath, destNetAddr, destAbsPath))
```

Comment:
@wangkx can you explain the difference between the old "name or address way" vs. the separate name and address way?  It might have been a little confusing but wouldn't looking at name first and then address still work?